isic/ingest/services/publish/__init__.py

```diff
@@ -94,7 +101,62 @@ def cohort_publish(*, publish_request: PublishRequest) -> None:
     sync_elasticsearch_indices_task.delay_on_commit()
 
 
+def embed_iptc_metadata(blob: FieldFile, attribution: str, license_: str) -> Path:
```

Comment:
Can you use a better name for `license_` without an arbitrary trailing underscore?